docs/src/site/twiki/FalconDocumentation.twiki

```diff
@@ -615,10 +617,35 @@ Failure to follow any of the above rules would result in a process submission fa
 present in the system for the specified time period, the process can be submitted and scheduled, but all instances
 created would result in a WAITING state unless data is actually provided in the cluster.
 
+---++ Updating cluster entity definition
+User would want to update the cluster entity in following scenarios
+   * when hadoop clusters are updated from unsecure to secure
+   * when hadoop clusters move from non high-availability to high-availability etc
 
+In these scenarios, user would want to change the cluster entity to reflect updated interface endpoints or properties.
+Updating cluster would require cascading update to feed and process entity jobs in the workflow engine. So Falcon restricts
+Cluster update to following
+   * Cluster update can only be performed by superuser and when Falcon Server is in safemode.
```

Comment:
How about: So Falcon only allows cluster update when:
1. Falcon server is in safemode;
2. The update is requested by superuser;
3. ...